### PR TITLE
Update tomcat.md

### PR DESCRIPTION
**Pull request by MaxiReglisse**
i modified some tricks in the tomcat configuration.

### CODE REVIEW

File: doc/setup/tomcat.md [hunk 1]
```diff
@@ -48,6 +48,7 @@ sudo keytool -importkeystore \
     -destkeystore /etc/tomcat6/keystore
 ```
 
+The srckeystore is "changeit" by default, and should be modified in /etc/default/cacerts.
```
Comment (fvanderbiest) [inline doc/setup/tomcat.md]: Is this sentence really useful ?

To my knowledge, keytool does not ask for this password

Comment (MaxiReglisse) [inline doc/setup/tomcat.md]: keytool asks for 2 passwords : one for srckeystore and the other for destkeystore.

Comment (fvanderbiest) [inline doc/setup/tomcat.md]: Funny, I do not remember this step ...

**Comment by fvanderbiest** [comment]
Cannot merge this PR. It has to be updated (eg through rebase or merge)

**Comment by MaxiReglisse** [comment]
root@georchestra:/etc/tomcat6# keytool -importkeystore \
>     -srckeystore /etc/ssl/certs/java/cacerts \
>     -destkeystore /etc/tomcat6/keystore
Saisissez le mot de passe du keystore de destination :  
Saisissez le mot de passe du keystore source :

**Pull request closed without merging**